src/Root.cpp

```diff
@@ -141,8 +141,14 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 
 	LOG("Reading server config...");
 
+	std::string conf_file = "settings.ini";
+	if (a_OverridesRepo->HasValue("Server","ConfigFile"))
+	{
+	conf_file = a_OverridesRepo->GetValue("Server","ConfigFile");
```

Comment:
Can you elaborate on the indentation?
Also, @madmaxoft i'm working on PluginManager